docs/3.0.0-beta.x/concepts/configurations.md

```diff
@@ -2,54 +2,190 @@
 sidebarDepth: 2
 ---
 
-# Configurations
+# Configuration
 
-The main configurations of the project are located in the `./config` directory. Additional configs can be added in the `./api/**/config` folder of each API and plugin by creating JavaScript or JSON files.
+Your application configuration lives in the `config` folder. All the configuration files are loaded on startup and can be accessed through the configuration provider.
 
-## Application
+When you have a file `./config/server.js` with the following config:
 
-Contains the main configurations relative to your project.
+```js
+module.exports = {
+  host: '0.0.0.0',
+};
+```
+
+You can access it as
+
+```js
+strapi.config.get('server.host', 'defaultValueIfUndefined');
```

Comment:
I have a question here.
Is it case sensitive?
`stra-pi.js` -> `strapi-pi.host`
`StRaPi.js` -> `StRaPi.host`

Reply:
it is case sensitive indeed.